src/writeData/components/WriteDataHelperBuckets.tsx

```diff
@@ -26,7 +26,11 @@ import {
 // Types
 import {AppState, ResourceType, Bucket} from 'src/types'
 
-const WriteDataHelperBuckets: FC = () => {
+interface Props {
```

Comment:
I'm handing in a classname so I can use this existing component but slightly tweak the styling to match the design, it's an optional prop so other instances of the component didn't need to be updated.

Reply:
Instead of passing in a flag, could we pass in the className itself? If a className is provided, use it. Otherwise, use the default one

Reply:
yes,  i'll update that